### PR TITLE
[3.x] Feature/change table names issue 79

### LINKED ISSUE #79: Feature: show Subject in Mailables table instead Namespace and Name
Users do not understand what namespace is, they understand the headline of the letter.

### PR DESCRIPTION
**Pull request by ReeceM**
## Description

This PR adds the option to have the subject displayed in the Mailables table. There is also a title hint left which has the value of the classname.

## Related Issue

Closes #79 

## Motivation and Context

UX

## How Has This Been Tested?

Tested, no side effects noticed in rendering.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] ~Breaking change (fix or feature that would cause existing functionality to change)~

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
~- [ ] All new and existing tests passed.~

### CODE REVIEW

**Comment by Qoraiche** [comment]
Thanks for the PR @ReeceM LGTM, just one thing: 
I think would be better if we make the display configured (e.g: display_subject).
also, did you test it with dynamic values?

**Comment by ReeceM** [comment]
Hi @Qoraiche, glad it looks good.

With the config option are you thinking something like:
```php
if (config('maileclipse.display_subject', false)) {
    /** @see \Illuminate\Mail\Mailable@buildSubject for the Str Functions, used to keep consistent. */
    $fqcns[$i]['subject'] = $mailable_data->subject ?? Str::title(Str::snake(class_basename($mailableClass), ' '));
}
```

I tested using the result from having the `->subject()` method called in the `build` function. Then also the `$subject` prop defined in the Mailable. Are those the ones specifically? :)

File: resources/views/sections/mailables.blade.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
                         <thead>
                             <tr>
                                 <th scope="col">{{ __('Name') }}</th>
-                                <th scope="col">{{ __('Namespace') }}</th>
+                                <th scope="col">{{ __('Subject') }}</th>
```
Comment (Qoraiche) [inline resources/views/sections/mailables.blade.php:34]: Add ability to switch subject/namespace (based on config value)

File: src/MailEclipse.php [hunk 2]
```diff
@@ -514,6 +514,8 @@ protected static function mailablesList()
                         $fqcns[$i]['markdown'] = self::getMarkdownViewName($mailable_data);
                         $fqcns[$i]['name'] = $name;
                         $fqcns[$i]['namespace'] = $mailableClass;
+                        /** @see \Illuminate\Mail\Mailable@buildSubject for the Str Functions, used to keep consistent. */
+                        $fqcns[$i]['subject'] = $mailable_data->subject ?? Str::title(Str::snake(class_basename($mailableClass), ' '));
```
Comment (Qoraiche) [inline src/MailEclipse.php:518]: Switch value based on config, Thanks!!

**Review by Qoraiche — CHANGES_REQUESTED** [review]

**Review by Qoraiche — APPROVED** [review]

**Pull request merged**